rest-server/src/models/job.js

```diff
@@ -132,98 +128,99 @@ class Job {
       });
   }
 
-  putJob(name, data, username, next) {
-    const originData = data;
+  putJob(name, data, originalData, username, next) {
```

Comment:
`data`, `originalData`, and `username` are duplicate.

Reply:
What should I do if I hope to save the original data (exactly what the user submitted) here?

Reply:
I'll abandon the previous PR (https://github.com/Microsoft/pai/pull/502). Let's discuss in this thread.

Reply:
please also revert #490 to remove `username` param

Reply:
set `username` in `data`, just use the previous implementation in https://github.com/Microsoft/pai/pull/490/files

Reply:
Got it. Thanks!